src/bout++.cxx

```diff
@@ -430,7 +430,14 @@ int bout_monitor(Solver *solver, BoutReal t, int iter, int NOUT) {
   // Data used for timing
   static bool first_time = true;
   static BoutReal wall_limit, mpi_start_time; // Keep track of remaining wall time
-
+  
+  static bool stopCheck;       // Check for file, exit if exists?
+  static string stopCheckName; // File checked, whose existence triggers a stop
```

Comment:
I forgot our policy on stdlib stuff (if we have one...) -- are we preferring new variables use `std::`?